docs/templating/layout/customize_storefront_layout.md

```diff
@@ -11,6 +11,24 @@ To customize your shop, you can override either whole templates, or specific com
 The built-in templates belong to the `storefront` [theme](design_engine.md).
 To override any of them, copy its directory structure in your template directory.
 
+## Customize with Twig Components
+
+You can customize parts of the storefront by using [Twig components](components.md).
+This allows you to inject your own widgets, extending the storefront behavior.
+
+The available groups for the storefront are:
+
+| Group name | Template file |
+|---|---|
+| `storefront-before-maincart` | `vendor/ibexa/storefront/src/bundle/Resources/views/themes/storefront/cart/component/maincart/maincart.html.twig` |
+| `storefront-after-maincart` | `vendor/ibexa/storefront/src/bundle/Resources/views/themes/storefront/cart/component/maincart/maincart.html.twig` |
+| `storefront-before-minicart` | `vendor/ibexa/storefront/src/bundle/Resources/views/themes/storefront/cart/component/minicart/minicart.html.twig` |
+| `storefront-after-minicart` | `vendor/ibexa/storefront/src/bundle/Resources/views/themes/storefront/cart/component/minicart/minicart.html.twig` |
+| `storefront-before-add-to-cart` | `vendor/ibexa/cart/src/bundle/Resources/views/themes/standard/cart/component/add_to_cart/add_to_cart.html.twig` |
```

Comment:
There would be a small change 

storefront-before-add-to-cart
storefront-after-add-to-cart
storefront-before-summary
storefront-after-summary

these would be used in different templates see following PR
https://github.com/ibexa/storefront/pull/236